moflask/flask.py

```diff
@@ -43,11 +43,13 @@ def init_defaults(self):
 
     def init_sentry(self):
         """Initialize Sentry."""
-        if any(key in self.config for key in ["SENTRY_DSN", "SENTRY_CONFIG"]):
-            # pylint: disable=import-outside-toplevel,import-error
-            from raven.contrib.flask import Sentry
+        # pylint: disable=import-outside-toplevel
+        if config := self.config.get("SENTRY_CONFIG"):
+            import sentry_sdk
+            from sentry_sdk.integrations.flask import FlaskIntegration
 
-            self.sentry = Sentry(self)
+            defaults = {"integrations": [FlaskIntegration()]}
+            self.sentry = sentry_sdk.Client(**{**defaults, **config})
```

Comment:
I think `sentry_sdk.Client(**defaults, **config)` would do the same?

Reply:
![image](https://user-images.githubusercontent.com/182122/225012608-8710c402-1038-4921-a3ca-552a78d5fe53.png)

Reply:
Ah, because `config` could override keywords from the `defaults` :bulb:

Reply:
Exactly ;-)